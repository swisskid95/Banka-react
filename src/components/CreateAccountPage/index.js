import React, { Component } from 'react';
import SideNavPanel from '../commons/SideNavPanel';
import TopSection from '../commons/TopSection';
import { connect } from 'react-redux';
import { createAccountRequest } from '../../redux/actions/createAccountActions';
import PropTypes from 'prop-types';
export class CreateAccountPage extends Component {
  state = {
    type: '',
    accountPurpose: '',
    isLoading: false
  };

  handleInputChange = event => {
    const { target } = event;
    this.setState({ [target.name]: target.value });
  };

  handleSubmit = async event => {
    event.preventDefault();
    const user = JSON.parse(localStorage.getItem('user'));
    this.setState({ isLoading: true });
    await this.props.createAccountRequest(user.token, this.state);
    this.setState({ isLoading: false });
  };

  render() {
    const { isLoading } = this.state;
    return (
      <main className="container">
        <SideNavPanel />
        <div className="main-wrapper">
          <TopSection />
          <div className="content-title">
            <h1 className="content-title__text">create account</h1>
          </div>
          <form onSubmit={this.handleSubmit} className="entry--closer entry">
            <select
              name="type"
              onBlur={this.handleInputChange}
              className="entry__input--account-type entry__input entry__input--bigger"
            >
              <option value="current">current</option>
              <option value="savings">savings</option>
            </select>
            <input
              type="text"
              onChange={this.handleInputChange}
              className="entry__input entry__input--bigger"
              name="accountPurpose"
              placeholder="account purpose"
            />
            <button
              className={`btn entry__btn entry__input--bigger ${
                isLoading ? 'btn-secondary' : ''
              }`}
            >
              {isLoading ? 'Loading...' : 'create account'}
            </button>
          </form>
        </div>
      </main>
    );
  }
}

CreateAccountPage.propTypes = {
  createAccountRequest: PropTypes.func.isRequired
};

export const mapDispatchToProps = dispatch => {
  return {
    createAccountRequest: async (token, data) =>
      dispatch(await createAccountRequest(token, data))
  };
};

export default connect(
  null,
  mapDispatchToProps
)(CreateAccountPage);
