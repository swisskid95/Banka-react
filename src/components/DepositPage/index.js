/* eslint-disable jsx-a11y/no-static-element-interactions */
/* eslint-disable jsx-a11y/click-events-have-key-events */
import React, { Component } from 'react';
import SideNavPanel from '../commons/SideNavPanel';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';
import TopSection from '../commons/TopSection';
import PageTitle from '../commons/PageTitle';
import {
  fetchAccountDetails,
  performDeposit
} from '../../redux/actions/cashierStaffAction';

export class DepositPage extends Component {
  state = {
    accountNumber: '',
    remarks: '',
    amount: '',
    isLoading: false,
    showModal: false
  };

  handleInputChange = event => {
    event.preventDefault();
    const { target } = event;
    this.setState({ [target.name]: target.value });
  };

  handleConfirmDetails = async event => {
    const { fetchAccountDetails, user } = this.props;
    event.preventDefault();
    this.setState({ isLoading: true });
    fetchAccountDetails(user.token, this.state.accountNumber);
    this.setState({ isLoading: false });
    this.setState({ showModal: true });
  };

  handleModalClose = () => {
    this.setState({ showModal: false });
  };

  render() {
    const { user, customerDetails, performDeposit } = this.props;
    const { accountNumber, remarks, amount, isLoading, showModal } = this.state;

    const handlePerformDeposit = async () => {
      this.setState({ isLoading: true });
      await performDeposit(user.token, accountNumber, { remarks, amount });
      this.setState({ isLoading: false });
      this.setState({ showModal: false });
    };
    return (
      <main className="container">
        <SideNavPanel user={user} />
        <div className="main-wrapper">
          <TopSection />
          <PageTitle name={'deposit'} />

          <form
            onSubmit={this.handleConfirmDetails}
            className="entry--closer entry"
          >
            <input
              type="number"
              className="entry__input entry__input--bigger"
              placeholder="account number"
              name="accountNumber"
              required
              onChange={this.handleInputChange}
              onBlur={this.handleInputChange}
            />
            <input
              type="number"
              className="entry__input entry-deposit__amount entry__input--bigger"
              placeholder="amount"
              name="amount"
              required
              onChange={this.handleInputChange}
              onBlur={this.handleInputChange}
            />
            <textarea
              className="entry__input entry__input--remark entry__input--bigger"
              placeholder="remark"
              name="remarks"
              required
              onChange={this.handleInputChange}
              onBlur={this.handleInputChange}
            ></textarea>
            <button
              disabled={isLoading}
              className="btn entry__btn entry__input--bigger modal__open"
            >
              deposit
            </button>
          </form>
        </div>

        <div
          onClick={this.handleModalClose}
          className={`modal ${showModal ? 'modal-open' : ''}`}
        >
          {!this.state.loading && Object.keys(customerDetails).length > 0 ? (
            <div href="#" className="transaction-full">
              <div className="transaction__details__title">
                <h1 className="content-title__text modal__title">
                  Confirm Deposit
                </h1>
              </div>
              <div className="transaction-full__main modal__main">
                <div className="transaction-full__type transaction-full__section">
                  <h4 className="transaction-full__title">Account Name</h4>
                  <p className="transaction-full__body">
                    {' '}
                    {`${customerDetails.accountDetails.owner.firstname} ${customerDetails.accountDetails.owner.lastname}`}
                  </p>
                </div>
                <div className="transaction-full__type transaction-full__section">
                  <h4 className="transaction-full__title">Account Number</h4>
                  <p className="transaction-full__body">
                    {customerDetails.accountDetails.accountnumber}
                  </p>
                </div>
                <div className="transaction-full__type transaction-full__section">
                  <h4 className="transaction-full__title">Amount</h4>
                  <p className="transaction-full__body">{amount}</p>
                </div>
                <div className="transaction-amount transaction-full__section">
                  <h4 className="transaction-full__title">remark</h4>
                  <p className="transaction-full__body">{remarks}</p>
                </div>
                <div className="transaction-full__section modal-response">
                  <button
                    className="btn modal__close action-center__btn btn-secondary"
                    onClick={this.handleModalClose}
                  >
                    cancel
                  </button>
                  <button
                    onClick={handlePerformDeposit}
                    className="btn action-center__btn"
                  >
                    confirm
                  </button>
                </div>
              </div>
            </div>
          ) : (
            'loading ...'
          )}
        </div>
      </main>
    );
  }
}

DepositPage.propTypes = {
  user: PropTypes.object,
  fetchAccountDetails: PropTypes.func,
  customerDetails: PropTypes.object,
  performDeposit: PropTypes.func
};

export const mapStateToProps = state => {
  return {
    user: state.users.userAuthDetails,
    customerDetails: state.customerDetails
  };
};

export const mapDispatchToProps = dispatch => ({
  fetchAccountDetails: (token, accountNumber) => {
    return dispatch(fetchAccountDetails(token, accountNumber));
  },
  performDeposit: (token, accountNumber, depositData) => {
    return dispatch(performDeposit(token, accountNumber, depositData));
  }
});

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(DepositPage);
