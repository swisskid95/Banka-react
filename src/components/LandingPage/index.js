import React from 'react';
import { Link } from 'react-router-dom';

const LandingPage = () => {
  return (
    <main className="container__entry">
      <div className="landing__page">
        <div className="section-wrapper">
          <section className="main-section">
            <h1 className="logo--big">Banka</h1>
            <h2 className="first-text">Banking made easier</h2>
            <p className="second-text">
              Banka helps you track all transaction on your account(s)
            </p>
            <Link to="/sign-up" className="cta">
              SIGN-UP{' '}
            </Link>
          </section>
          <section className="infograph-section">
            <img
              className="monitor"
              src="../../src/assets/images/infograph.svg"
              alt="infograph"
            />
          </section>
        </div>
      </div>
    </main>
  );
};

export default LandingPage;
