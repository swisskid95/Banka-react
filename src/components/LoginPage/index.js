/* eslint-disable no-console */
import React, { Component } from 'react';
import EntrySideBar from '../commons/EntrySideBar';
import TopSection from '../commons/TopSection';
import PageTitle from '../commons/PageTitle';
import { Link } from 'react-router-dom';
import PropTypes from 'prop-types';
import { connect } from 'react-redux';
import { loginUserRequest } from '../../redux/actions/loginActions';

export class LoginPage extends Component {
  state = {
    email: '',
    password: '',
    isLoading: false
  };

  handleInputChange = event => {
    event.preventDefault();
    const { target } = event;

    this.setState({ [target.name]: target.value });
  };

  handleSubmit = async event => {
    event.preventDefault();
    this.setState({ isLoading: true });
    await this.props.loginUserRequest(this.state, this.props.history);
    this.setState({ isLoading: false });
  };

  render() {
    const { isLoading } = this.state;
    return (
      <main className="container__entry">
        <EntrySideBar
          welcomeText="HELLO, Friend"
          mainText="Not yet a member, click on the button below to Join in the benefits."
          buttonText="sign-up"
        />
        <TopSection />
        <PageTitle name={'login'} />
        <form className="entry" onSubmit={this.handleSubmit}>
          <div className="field">
            <input
              className="entry__input"
              type="email"
              placeholder="Email"
              name="email"
              required
              onChange={this.handleInputChange}
            />
            <span className="helper-text"></span>
          </div>
          <div className="field">
            <input
              className="entry__input"
              type="password"
              placeholder="Password"
              name="password"
              required
              onChange={this.handleInputChange}
            />
            <span className="helper-text"></span>
          </div>
          <button
            className={`entry__btn btn ${isLoading ? 'btn-secondary' : ''}`}
          >
            {isLoading ? 'Loading...' : 'Login'}
          </button>
        </form>
        <div className="switch">
          <p>
            Not yet a member?
            <Link to="/sign-up" className="switch__link">
              sign-up
            </Link>
          </p>
        </div>
      </main>
    );
  }
}

LoginPage.propTypes = {
  loginUserRequest: PropTypes.func,
  history: PropTypes.any
};

export const mapDispatchToProps = dispatch => {
  return {
    loginUserRequest: async (loginCredentials, history) =>
      dispatch(await loginUserRequest(loginCredentials, history))
  };
};

export default connect(
  null,
  mapDispatchToProps
)(LoginPage);
