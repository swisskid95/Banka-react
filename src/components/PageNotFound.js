import React from 'react';
import TopSection from '../components/commons/TopSection';

const PageNotFound = () => (
  <main className="container">
    <div className="main-wrapper">
      <TopSection />
      <h1 className="page-not-found">Oops! Page not found.</h1>
    </div>
  </main>
);

export default PageNotFound;
