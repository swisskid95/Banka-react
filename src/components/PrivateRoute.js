/* istanbul ignore file */
import React from 'react';
import { Route, Redirect } from 'react-router-dom';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';

const PrivateRoute = ({ component: Component, auth, ...rest }) => (
  <React.Fragment>
    <Route
      {...rest}
      render={props =>
        auth.isAuthenticated === true ? (
          <Component {...props} />
        ) : (
          <Redirect to="/login" />
        )
      }
    />
  </React.Fragment>
);

PrivateRoute.propTypes = {
  auth: PropTypes.object.isRequired,
  component: PropTypes.object.isRequired
};

const mapStateToProps = state => ({
  auth: state.users
});

export default connect(mapStateToProps)(PrivateRoute);
