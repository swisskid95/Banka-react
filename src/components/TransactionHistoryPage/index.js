import React from 'react';
import SideNavPanel from '../commons/SideNavPanel';

export const TransactionHistoryPage = () => {
  return (
    <main className="container">
      <SideNavPanel />
      <div className="main-wrapper">
        <div className="top-section">
          <img
            src="../../src/assets/images/menu.svg"
            alt="Sidebar open button"
            className="sidebar__open"
          />
          <h1 className="logo">Banka</h1>
        </div>

        <div className="content-title">
          <h1 className="content-title__text">transaction history</h1>
        </div>
        <div className="summary-field summary-field--mini">
          <img
            className="summary__nav summary__nav--mini"
            src="../../src/assets/images/left-arrow.svg"
            alt="left arrow navigator"
          />
          <div className="summary summary--mini">
            <div className="account-type">
              <p className="summary__text--mini">current account</p>
            </div>
            <div className="account-number">
              <p className="summary__text--mini">0005894789</p>
            </div>
            <div className="summary__bal--mini">
              &#8358;
              <span>45,000.00</span>
            </div>
          </div>
          <img
            className="summary__nav summary__nav--mini"
            src="../../src/assets/images/right-arrow.svg"
            alt="right arrow navigator"
          />
        </div>
        <form className="entry-history">
          <input
            placeholder="from"
            name="from"
            type="date"
            className=" input-from entry-history__input"
          />
          <input
            placeholder="to"
            name="to"
            type="date"
            className="input-to entry-history__input"
          />
          <button className="entry-history__btn btn">filter with date</button>
        </form>
        <div className="transaction modal__open">
          <div className="transaction__col1">
            <p className="transaction__amount">45,000.00</p>
            <p className="transaction__date">Mon, 12-Mar-2019</p>
          </div>
          <div className="transaction__col2">
            <p className="transaction__details">
              Withdrawal at Banka branch Alakija branch Ibadan-Express way off
              Hunt-hill close surulere Lagos.
            </p>
          </div>
        </div>
        <div className="transaction modal__open">
          <div className="transaction__col1">
            <p className="transaction__amount">45,000.00</p>
            <p className="transaction__date">Mon, 12-Mar-2019</p>
          </div>
          <div className="transaction__col2">
            <p className="transaction__details">
              Withdrawal at Banka branch Alakija branch Ibadan-Express way off
              Hunt-hill close surulere Lagos.
            </p>
          </div>
        </div>
        <div className="transaction modal__open">
          <div className="transaction__col1">
            <p className="transaction__amount">45,000.00</p>
            <p className="transaction__date">Mon, 12-Mar-2019</p>
          </div>
          <div className="transaction__col2">
            <p className="transaction__details">
              Withdrawal at Banka branch Alakija branch Ibadan-Express way off
              Hunt-hill close surulere Lagos.
            </p>
          </div>
        </div>
        <div className="transaction modal__open">
          <div className="transaction__col1">
            <p className="transaction__amount">45,000.00</p>
            <p className="transaction__date">Mon, 12-Mar-2019</p>
          </div>
          <div className="transaction__col2">
            <p className="transaction__details">
              Withdrawal at Banka branch Alakija branch Ibadan-Express way off
              Hunt-hill close surulere Lagos.
            </p>
          </div>
        </div>
      </div>

      <div className="modal">
        <div className="transaction-full no-sidebar-page">
          <div className="transaction__details__title">
            <h1 className="content-title__text modal__title">
              transaction details
            </h1>
            <span className="modal__close transaction--close">&times;</span>
          </div>
          <div className="transaction-full__main">
            <div className="transaction-full__type transaction-full__section">
              <h4 className="transaction-full__title">transaction type</h4>
              <p className="transaction-full__body">Debit</p>
            </div>
            <div className="transaction-full__type transaction-full__section">
              <h4 className="transaction-full__title">transaction date</h4>
              <p className="transaction-full__body">Mon, 12-Mar-2019</p>
            </div>
            <div className="transaction-amount transaction-full__section">
              <h4 className="transaction-full__title">amount</h4>
              <p className="transaction-full__body">45,000.00</p>
            </div>
            <div className="transaction__remark transaction-full__section">
              <h4 className="transaction-full__title">old balance</h4>
              <p className="transaction-full__body">195,789</p>
            </div>
            <div className="transaction__remark transaction-full__section">
              <h4 className="transaction-full__title">new Balance</h4>
              <p className="transaction-full__body">150,700</p>
            </div>
            <div className="transaction__remark transaction-full__section">
              <h4 className="transaction-full__title">remark</h4>
              <p className="transaction-full__body">
                Withdrawal at Banka branch Alakija branch Ibadan-Express way off
                Hunt-hill close surulere Lagos.
              </p>
            </div>
          </div>
        </div>
      </div>
    </main>
  );
};

export default TransactionHistoryPage;
