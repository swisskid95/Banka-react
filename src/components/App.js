import React from 'react';
import { Route, Switch, BrowserRouter as Router } from 'react-router-dom';
import { hot } from 'react-hot-loader';
import PageNotFound from './PageNotFound';
import LandingPage from './LandingPage';
import SignupPage from './SignupPage';
import LoginPage from './LoginPage';
import SummaryPage from './SummaryPage';
import CreateAccountPage from './CreateAccountPage';
import { ToastContainer } from 'react-toastify';
import DepositPage from './DepositPage';
import WithdrawalPage from './WithdrawalPage';
import PrivateRoute from './PrivateRoute';

class App extends React.Component {
  render() {
    return (
      <Router>
        <ToastContainer />
        <Switch>
          <Route exact path="/" component={LandingPage} />
          <Route path="/sign-up" component={SignupPage} />
          <Route path="/login" component={LoginPage} />
          <PrivateRoute path="/summary" component={SummaryPage} />
          <PrivateRoute path="/create-account" component={CreateAccountPage} />
          <PrivateRoute path="/deposit" component={DepositPage} />
          <PrivateRoute path="/withdraw" component={WithdrawalPage} />
          <Route component={PageNotFound} />
        </Switch>
      </Router>
    );
  }
}

export default hot(module)(App);
