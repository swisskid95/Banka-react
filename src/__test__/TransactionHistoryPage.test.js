/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import { TransactionHistoryPage } from '../components/TransactionHistoryPage';

describe('TransactionHistoryPage', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<TransactionHistoryPage />);

    expect(wrapper).toMatchSnapshot();
  });
});
