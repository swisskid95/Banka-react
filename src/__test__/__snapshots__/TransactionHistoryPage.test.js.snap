// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransactionHistoryPage should render correctly 1`] = `
<main
  className="container"
>
  <SideNavPanel />
  <div
    className="main-wrapper"
  >
    <TopSection />
    <PageTitle
      name="transaction history"
    />
    <div
      className="summary-field summary-field--mini"
    >
      <img
        alt="left arrow navigator"
        className="summary__nav summary__nav--mini"
        src="https://res.cloudinary.com/swisskid95/image/upload/v1566575801/Banka-assets/left-arrow_j7rkn3.svg"
      />
      <div
        className="summary summary--mini"
      >
        <div
          className="account-type"
        >
          <p
            className="summary__text--mini"
          >
            current account
          </p>
        </div>
        <div
          className="account-number"
        >
          <p
            className="summary__text--mini"
          >
            0005894789
          </p>
        </div>
        <div
          className="summary__bal--mini"
        >
          ₦
          <span>
            45,000.00
          </span>
        </div>
      </div>
      <img
        alt="right arrow navigator"
        className="summary__nav summary__nav--mini"
        src="https://res.cloudinary.com/swisskid95/image/upload/v1566575801/Banka-assets/right-arrow_w5gibu.svg"
      />
    </div>
    <div
      className="transaction modal__open"
    >
      <div
        className="transaction__col1"
      >
        <p
          className="transaction__amount"
        >
          45,000.00
        </p>
        <p
          className="transaction__date"
        >
          Mon, 12-Mar-2019
        </p>
      </div>
      <div
        className="transaction__col2"
      >
        <p
          className="transaction__details"
        >
          Withdrawal at Banka branch Alakija branch Ibadan-Express way off Hunt-hill close surulere Lagos.
        </p>
      </div>
    </div>
    <div
      className="transaction modal__open"
    >
      <div
        className="transaction__col1"
      >
        <p
          className="transaction__amount"
        >
          45,000.00
        </p>
        <p
          className="transaction__date"
        >
          Mon, 12-Mar-2019
        </p>
      </div>
      <div
        className="transaction__col2"
      >
        <p
          className="transaction__details"
        >
          Withdrawal at Banka branch Alakija branch Ibadan-Express way off Hunt-hill close surulere Lagos.
        </p>
      </div>
    </div>
    <div
      className="transaction modal__open"
    >
      <div
        className="transaction__col1"
      >
        <p
          className="transaction__amount"
        >
          45,000.00
        </p>
        <p
          className="transaction__date"
        >
          Mon, 12-Mar-2019
        </p>
      </div>
      <div
        className="transaction__col2"
      >
        <p
          className="transaction__details"
        >
          Withdrawal at Banka branch Alakija branch Ibadan-Express way off Hunt-hill close surulere Lagos.
        </p>
      </div>
    </div>
    <div
      className="transaction modal__open"
    >
      <div
        className="transaction__col1"
      >
        <p
          className="transaction__amount"
        >
          45,000.00
        </p>
        <p
          className="transaction__date"
        >
          Mon, 12-Mar-2019
        </p>
      </div>
      <div
        className="transaction__col2"
      >
        <p
          className="transaction__details"
        >
          Withdrawal at Banka branch Alakija branch Ibadan-Express way off Hunt-hill close surulere Lagos.
        </p>
      </div>
    </div>
  </div>
  <div
    className="modal"
  >
    <div
      className="transaction-full no-sidebar-page"
    >
      <div
        className="transaction__details__title"
      >
        <h1
          className="content-title__text modal__title"
        >
          transaction details
        </h1>
        <span
          className="modal__close transaction--close"
        >
          ×
        </span>
      </div>
      <div
        className="transaction-full__main"
      >
        <div
          className="transaction-full__type transaction-full__section"
        >
          <h4
            className="transaction-full__title"
          >
            transaction type
          </h4>
          <p
            className="transaction-full__body"
          >
            Debit
          </p>
        </div>
        <div
          className="transaction-full__type transaction-full__section"
        >
          <h4
            className="transaction-full__title"
          >
            transaction date
          </h4>
          <p
            className="transaction-full__body"
          >
            Mon, 12-Mar-2019
          </p>
        </div>
        <div
          className="transaction-amount transaction-full__section"
        >
          <h4
            className="transaction-full__title"
          >
            amount
          </h4>
          <p
            className="transaction-full__body"
          >
            45,000.00
          </p>
        </div>
        <div
          className="transaction__remark transaction-full__section"
        >
          <h4
            className="transaction-full__title"
          >
            old balance
          </h4>
          <p
            className="transaction-full__body"
          >
            195,789
          </p>
        </div>
        <div
          className="transaction__remark transaction-full__section"
        >
          <h4
            className="transaction-full__title"
          >
            new Balance
          </h4>
          <p
            className="transaction-full__body"
          >
            150,700
          </p>
        </div>
        <div
          className="transaction__remark transaction-full__section"
        >
          <h4
            className="transaction-full__title"
          >
            remark
          </h4>
          <p
            className="transaction-full__body"
          >
            Withdrawal at Banka branch Alakija branch Ibadan-Express way off Hunt-hill close surulere Lagos.
          </p>
        </div>
      </div>
    </div>
  </div>
</main>
`;
